java-checks/src/main/java/org/sonar/java/checks/helpers/RegexTreeHelper.java

```diff
@@ -30,7 +30,7 @@
 import org.sonar.java.regex.ast.AutomatonState;
 import org.sonar.java.regex.ast.AutomatonState.TransitionType;
 import org.sonar.java.regex.ast.BoundaryTree;
-import org.sonar.java.regex.ast.CharacterTree;
+import org.sonar.java.regex.ast.PlainCharacterTree;
```

Comment:
Given the amount of changes like this, it might make more sense to rename `PlainCharacterTree` to `CharacterTree` (and the corresponding enum value to `CHARACTER`). The qualifier "plain" isn't really necessary anymore since there's no other type of character tree to distinguish it from. Plus, since previously "plain" meant "single code unit", it's no longer accurate anyway and was confusing nomenclature to begin with.